inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -171,6 +172,7 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
             manager.register_pass<ngraph::pass::ConvertNMS4ToNMS5>();
             manager.register_pass<ngraph::pass::ConvertNMSToNMSIEInternal>();
             manager.register_pass<ngraph::pass::ConvertGather0D>();
+            manager.register_pass<ngraph::pass::SimplifyCTCGreedyDecoder>();
```

Comment:
I think that we should invoke this transformation for plugins which doesn't support `CTCGreedyDecoderSeqLen` currently.

Reply:
We need enable replacer for each plugin where op not merged yet.